core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -93,6 +93,7 @@ interface CreatorDashboardData {
   providedIn: 'root'
 })
 export class CreatorDashboardBackendApiService {
+  _fetchDashboardData: any;
```

Comment:
Why is this needed?

Reply:
Apologies. I learned that it is not needed. I will change it. Thanks!